promptshell/main.py

```diff
@@ -53,12 +55,67 @@ def main():
 - Type 'clear' to clear the terminal.{reset_format()}""")
                 continue
 
+            if user_input.lower().startswith("alias "):
+                result = handle_alias_command(user_input, assistant.alias_manager)
+                print(result)
+                continue
+
             result = assistant.execute_command(user_input)
             print(result)
 
         except KeyboardInterrupt:
             print(format_text('red', bold=True) + "\nTerminating..." + reset_format())
             break
 
+def handle_alias_command(command: str, alias_manager: AliasManager) -> str:
```

Comment:
```suggestion
def handle_alias_command(command: str, alias_manager: AliasManager) -> str: (move it to alias_manager.py)
```
move this function to alias_manager.py for better separation of concerns and handle the imports accordingly